src/Common/tests/System/PlatformDetection.cs

```diff
@@ -34,6 +34,58 @@ public static class PlatformDetection
 
         public static int WindowsVersion { get; } = GetWindowsVersion();
 
+        private static int s_isWinRT = -1;
```

Comment:
You'd think so at first, but it is unfortunately. The [contructor](http://source.dot.net/#System.Private.CoreLib/src/System/Nullable.cs,31), for example, can actually have its instructions reordered to where it says it is set when it isn't. So reading it will tell you it is set and it still has the default value, causing a verification failure in the test.